glfw.go

```diff
@@ -105,9 +105,17 @@ func (m *windowManager) glfwCursorEnterCallback(window *glfw.Window, entered boo
 	x, y := window.GetCursorPos()
 	if entered {
 		m.sendPointerEvent(window, embedder.PointerPhaseAdd, x, y)
-		m.pointerPhase = embedder.PointerPhaseHover
+		// the mouse can enter the windows while having button pressed.
+		// if so, don't overwrite the phase.
+		if m.pointerButton == 0 {
+			m.pointerPhase = embedder.PointerPhaseHover
+		}
 	} else {
-		m.sendPointerEvent(window, embedder.PointerPhaseRemove, x, y)
+		// if the mouse is still in 'phaseMove' outside the window (click-drag
+		// outside). Don't remove the cursor.
+		if m.pointerButton == 0 {
+			m.sendPointerEvent(window, embedder.PointerPhaseRemove, x, y)
```

Comment:
What if the mouse is released outside the window? Do we need to send a remove at that time?

Reply:
We don't have to schedule a `PointerPhaseRemove` because `glfwCursorEnterCallback` gets called when the mouse is dragged outside AND when the outside mouse is released.

Reply:
![Kazam_screencast_00000](https://user-images.githubusercontent.com/7476655/61521323-437d6200-aa10-11e9-9599-463cf6eec186.gif)